clients/web/src/views/widgets/MetadataWidget.js

```diff
@@ -13,7 +13,10 @@ girder.views.MetadataWidget = girder.View.extend({
 
     initialize: function (settings) {
         this.item = settings.item;
+        this.fieldName = settings.fieldName || 'meta';
         this.accessLevel = settings.accessLevel;
+        this.onMetadataEdited = settings.onMetadataEdited;
```

Comment:
Nevermind, ``girder.views.MetadatumEditWidget`` is part of our public API surface (although most people would not instantiate it directly), so we have to support it being created without the ``onMetadataEdited`` setting.